spec/fixtures/user/confirmation

```diff
@@ -0,0 +1,3 @@
+User#confirmation
+
+Hi, find me in app/views/user/confirmation
```

Comment:
🙄 🤣

Reply:
I generated classes automatically when I used the command line to generate them.
I removed it. 
Thank you!